tests/aggregation_regress/tests.py

```diff
@@ -360,6 +361,40 @@ def test_aggrate_annotation(self):
             {'c__max': 3}
         )
 
+    def test_conditional_aggregate_of_subquery(self):
+        """
+        Regression test for #25307: AttributeError: 'WhereNode' object has no
+        attribute 'get_source_expressions'.
+
+        Tests conditional aggregations over subqueries.
+        """
+        # Conditional aggregation on a limited queryset.
```

Comment:
`test_sliced_conditionnal_aggregate()`

Did this also fail without the patch applied? I see no mentions of it on the ticket.